src/constants.js

```diff
@@ -0,0 +1,12 @@
+module.exports = Object.freeze({
+    PROPERTIES: {
+        any: 'LoadState,ActiveState,SubState,Description,Id,Result,UnitFileState,ActiveEnterTimestamp,ActiveEnterTimestampMonotonic',
+        service: 'CleanResult,MainPID,Type,Restart,MemoryCurrent,Nice,RemainAfterExit,TasksCurrent',
+        mount: 'Where,What,Type,LazyUnmount',
+        slice: 'TasksCurrent'
+        //target: '',
```

Comment:
Any reason these are commented out, but still there?

Reply:
Because I want to collect every type of systemd unit for easier editing.